src/sidebar/util/user-agent.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Helper methods to identify browser versions and os types
+ */
+
+export const isIE11 = (_window = window) => {
+  return _window.navigator.userAgent.indexOf('rv:11') >= 0;
```

Comment:
This test might be overly generic. If we have test for IE specifically, I would probably rather test for the `Trident/7.0` part, since "Trident" is the codename for IE's browsing engine.

Reply:
I agree